main.go

```diff
@@ -112,3 +121,24 @@ Arguments:
 		helpTab,
 	)
 }
+
+func relativeWindowsPath(dir string) string {
+	fmt.Printf("Received '%s'\n", dir)
+	usr, err := user.Current()
+	if err != nil {
+		fmt.Printf("Could not get user: %v", err)
+		return dir
+	}
+	rel, err := filepath.Rel(usr.HomeDir, dir)
+	if err != nil {
+		return dir
+	}
+	rel = "~/" + filepath.ToSlash(rel)
+	return rel
+}
+
+func gitbashWindowsDir(dir string) (res string) {
```

Comment:
Could you add a comment stating why this is required? Something along the lines of:
```
// gitbashWindowsDir translates a directory similar to `C:\Users\username\path` to `~/path` for compatibility with Git bash.
```